dataflow/src/main/java/org/checkerframework/dataflow/busyexpression/BusyExprValue.java

```diff
@@ -0,0 +1,49 @@
+package org.checkerframework.dataflow.busyexpression;
+
+import org.checkerframework.checker.nullness.qual.Nullable;
+import org.checkerframework.dataflow.analysis.AbstractValue;
+import org.checkerframework.dataflow.cfg.node.Node;
+import org.checkerframework.javacutil.BugInCF;
+
+/** A busy expression wrapper turning a node into the abstract value */
+public class BusyExprValue implements AbstractValue<BusyExprValue> {
+
+    /**
+     * A busy expression is represented by a node, which can be a {@link
+     * org.checkerframework.dataflow.cfg.node.BinaryOperationNode}
+     */
+    protected final Node busyExpression;
+
+    @Override
+    public BusyExprValue leastUpperBound(BusyExprValue other) {
+        throw new BugInCF("lub of BusyExpValue get called!");
+    }
+
+    /**
+     * Create a new busy expression.
+     *
+     * @param n a node
+     */
+    public BusyExprValue(Node n) {
```

Comment:
The usual order puts the constructor before methods.
(I see that these problems also exist in the live var demo. Can you make a separate PR to clean that up?)

Reply:
Yes sure.